wscript

```diff
@@ -778,6 +778,16 @@ video_output_features = [
         'name': '--sixel',
         'desc': 'Sixel',
         'func': check_pkg_config('libsixel', '>= 1.5'),
+    }, {
+        'name': 'dmabuf-interop-gl',
+        'desc': 'dmabuf GL Interop',
+        'deps': 'vaapi-egl && drm',
+        'func': check_true,
```

Comment:
another thought: why have these options at all? you can use `HAVE_EGL && HAVE_DRM` in the code

Reply:
I felt that with `hwdec=drmprime`, it will be clearer to have distinct config options for these.

Reply:
Was just wondering, sounds okay.